stix_shifter/stix_translation/src/modules/security_advisor/json/to_stix_map.json

```diff
@@ -0,0 +1,80 @@
+{
+
+  "author_id": {
+    "key": "author.user_id"
+  },
+
+  "author_account_id": {
+    "key": "author.author_account_id"
+  },
+
+  "author_email": {
+    "key": "author.email_id"
```

Comment:
Is this an actual email address? If so it should be mapped to `email-addr.value`

Reply:
Done